src/scripts/components/dialog/edit-board.js

```diff
@@ -23,23 +23,49 @@ export default React.createClass({
 		return {
 			name:             this.props.board.name,
 			background:       this.props.board.background,
-			customBackground: this.props.board.customBackground
+			customBackground: this.props.board.customBackground,
+			width:            this.props.board.size.width,
+			height:           this.props.board.size.height
 		}
 	},
 
 	submit(event) {
 		event.preventDefault();
 
-		BoardAction.update({
-			id:               this.props.board.id,
-			name:             this.state.name,
-			background:       this.state.background,
-			customBackground: this.state.customBackground
-		});
+		let size = {
+			"width": this.state.width,
+			"height": this.state.height
+		};
+
+
+		if (!isNaN(size.width) && !isNaN(size.height)) {
```

Comment:
Why do we need to check for `NaN` values here? Can't we do that in the form itself?